Tests/ScipioKitTests/RunnerTests.swift

```diff
@@ -439,6 +446,8 @@ final class RunnerTests: XCTestCase {
     }
 
     func testWithPlatformMatrix() async throws {
+        try skipIfCIIsNotSupported()
```

Comment:
Please write why we need to skip it in comment

Reply:
I added comment